package.json

```diff
@@ -1,16 +1,22 @@
 {
-  "name": "vivy-components",
-  "version": "0.0.5",
-  "private": true,
+  "name": "@vivy/vivy-components",
+  "version": "0.0.6",
```

Comment:
should we update it automatically?

Reply:
We'll have to update it manually anyways because even if we did increment it we'd need to commit the changes to git which will then retrigger master and cause an infinite loop.

Reply:
mb filter it with commit message or trigger explicitly with tag?

Reply:
Sorry, I don't follow

Reply:
We might be able to skip builds based on commit message - but if we don’t tag the build (or increment the package) the build itself will fail and not be published. We can check if it’s been changed but we’ll have to do it in code.

Reply:
what I'm worrying about is that we can forget to change the version here, or it can be inconvenient to do it manually as requires commit, where tag can be created using github release interface